ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-release-4.18.yaml

```diff
@@ -0,0 +1,194 @@
+promotion:
+  to:
+  - name: numaresources-operator-4.18
+    namespace: ocp-kni
+releases:
+  initial:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.18"
+resources:
+  '*':
+    requests:
+      cpu: 150m
+      memory: 250Mi
+tests:
+- as: ci-unit
+  commands: make test-unit
+  container:
+    from: src
+- as: ci-install-e2e
+  steps:
+    cluster_profile: aws-telco
+    dependencies:
+      OO_BUNDLE: tested-numaresources-operator-bundle
+    env:
+      BASE_DOMAIN: telco5g-ci.devcluster.openshift.com
+      OO_INSTALL_MODE: OwnNamespace
+      OO_INSTALL_NAMESPACE: numaresources-operator
+      USER_TAGS: |
+        ci-job-source prowci
+        ci-job-branch release-4.18
+        ci-job-fullname openshift-kni-ci-install-e2e-numaresources-operator-release-4.18
+        ci-job-type ci-install-e2e
+        ci-repo-name numaresources-operator
+        ci-org-name openshift-kni
+    test:
+    - as: e2e-install-test
+      cli: latest
+      commands: make test-install-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    workflow: optional-operators-ci-operator-sdk-aws
+- as: ci-e2e
+  steps:
+    cluster_profile: aws-telco
+    dependencies:
+      OO_BUNDLE: tested-numaresources-operator-bundle
+    env:
+      BASE_DOMAIN: telco5g-ci.devcluster.openshift.com
+      OO_INSTALL_MODE: OwnNamespace
+      OO_INSTALL_NAMESPACE: openshift-numaresources
+      USER_TAGS: |
+        ci-job-source prowci
+        ci-job-branch release-4.18
+        ci-job-fullname openshift-kni-ci-e2e-numaresources-operator-release-4.18
+        ci-job-type ci-e2e
+        ci-repo-name numaresources-operator
+        ci-org-name openshift-kni
+    test:
+    - as: e2e-test
+      cli: latest
+      commands: make test-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    workflow: optional-operators-ci-operator-sdk-aws
+- as: ci-must-gather-e2e
+  cron: 42 0 * * *
+  steps:
+    cluster_profile: aws-telco
+    dependencies:
+      OO_BUNDLE: tested-numaresources-operator-bundle
+    env:
+      BASE_DOMAIN: telco5g-ci.devcluster.openshift.com
+      OO_INSTALL_MODE: OwnNamespace
+      OO_INSTALL_NAMESPACE: openshift-numaresources
+      USER_TAGS: |
+        ci-job-source prowci
+        ci-job-branch release-4.18
+        ci-job-fullname openshift-kni-ci-must-gather-e2e-numaresources-operator-release-4.18
+        ci-job-type ci-must-gather-e2e
+        ci-repo-name numaresources-operator
+        ci-org-name openshift-kni
+    test:
+    - as: must-gather-e2e-test
+      cli: latest
+      commands: make test-must-gather-e2e
+      from: src
+      resources:
+        requests:
+          cpu: 500m
+          memory: 1Gi
+    workflow: optional-operators-ci-operator-sdk-aws
+- as: ci-e2e-install-hypershift
```

Comment:
Note: starting 4.18 we need to run hypershift CI, so inheriting that from main file